task/01-strings-tasks.js

```diff
@@ -201,7 +201,22 @@ function extractEmails(str) {
  *
  */
 function getRectangleString(width, height) {
-    throw new Error('Not implemented');
+    const shape = {
+        leftTopCorner: '┌',
+        rightTopCorner: '┐',
+        leftBottomCorner: '└',
+        rightBottomCorner: '┘',
+        horizontalLine: '─',
+        verticalLine: '│',
+        lineBreak: '\n',
+        space: ' ',
+    };
+
+    let top = `${shape.leftTopCorner}${shape.horizontalLine.repeat(width - 2)}${shape.rightTopCorner}${shape.lineBreak}`;
```

Comment:
Please replace every let with a `const`. Always use `const` and only when necessary use `let`